### PR TITLE
Feature Milestone 1: Setup Project

### PR DESCRIPTION
**Pull request by Div685**
> ### This Milestone focused on Setting up the Project
In this PR, I have completed the following requirements:
- Added ESLint and style Linters
- Deleted unnecessary files and assets.
- Added prop-types and Redux dependencies.
- Created Heroku app and hosted it on Heroku.

### CODE REVIEW

**Review by arslanbisharat — APPROVED** [review]
# Approved :clap:

Hello Developer :wave:

Thank You for submitting this Pull Request. I am thrilled to inform you that your project is approved now. :clap:

You have done a great job. I like your project. 

**To highlight** 

- [x] You have a descriptive Readme. Nice Job. 
- [x] Your Pull Request is nice. Good Job!
- [x] You have deployed your website. Nice!!!
- [x] You are using linters and no error is present. I am happy to see it. 
- [x] Your project has met the minimum technical requirements too. 



In short, I am very excited to see this project and proud of you. 

**What to do next?** :thinking_face:

1- Visit https://dashboard.microverse.org/projects and submit your approved project. 
2- Prepare yourself for the next project. 

I am wishing you very good luck with the next project. :smile:

Once again, Congratulations!:joy:

**Best,
Arslan Bisharat(PT-TSE)
Slack: @arslanbisharat**
